extensions/skins/conversation_v1/Conversation.js

```diff
@@ -299,6 +315,8 @@ oppia.directive('conversationSkin', [function() {
       var _navigateToCard = function(index) {
         $scope.activeCard = $scope.transcript[index];
         $scope.arePreviousResponsesShown = false;
+        $scope.clearHelpHtml();
+        $scope.clearContinueToNextCardHtml();
```

Comment:
For example, this would become:

$scope.helpHtml = null;
$scope.contineToNextCard = false;